concordium-consensus/src/Concordium/GlobalState/ContractStateV1.hs

```diff
@@ -186,10 +182,6 @@ instance (MonadBlobStore m) => BlobStorable m PersistentState where
       return (put bRef, ps)
 
 instance MonadBlobStore m => Cacheable m PersistentState where
-  cache ps = do
```

Comment:
Perhaps add a comment here why we are using a default "no-op" instance here.

Reply:
Sounds like a good idea!